wiki/Game_Modes/External_Ports/T-Aiko!/Interface/en.md

```diff
@@ -0,0 +1,125 @@
+<!-- If possible, use a script to do this, mate. Pardon the terrible naming sense. ~RaikireHiuduo -->
```

Comment:
Linebreak, or the single column table used in osu!tourney?

:thinking: 

Styling issue I suppose. Will be skipping this.